servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRestNamespace.java

```diff
@@ -55,8 +56,16 @@ public void testNamespaces(String namespaceName) throws BaseNessieClientServerEx
     Namespace namespace =
         getApi().createNamespace().refName(branch.getName()).namespace(ns).create();
 
-    assertThat(namespace).isNotNull().isEqualTo(ns);
-    assertThat(getApi().getNamespace().refName(branch.getName()).namespace(ns).get())
+    assertThat(namespace)
+        .isNotNull()
+        .extracting(Namespace::getElements, Namespace::toPathString)
+        .containsExactly(ns.getElements(), ns.toPathString());
+
+    Namespace got = getApi().getNamespace().refName(branch.getName()).namespace(ns).get();
+    assertThat(got)
+        .describedAs(
```

Comment:
nit: do we need a custom description here? I suppose the default `Namespace.toString` would be clear in terms of mismatching fields... WDYT?